### ISSUE TITLE
Thingy compatible code

### ISSUE THREAD

**Issue by SamGeens** [MOSAIC-LoPoW/octa-stack-students#48]
We wrote some custom code which is working on the development kit with pca10056/s140 and on the octa with pca10040/s132. Now we would like to flash this code to our thingies. But a Thingy seems kinda out of date, with a [deprecated thingy sdk](https://github.com/NordicSemiconductor/Nordic-Thingy52-FW) and with sdk13. 

The thingy should be compatible with softdevice s132 [(source)](https://nordicsemiconductor.github.io/Nordic-Thingy52-FW/) and should use board pca20020. This board file is available in the new SDK17 under components/boards. I created a pca20020/s132 directory in the project and flashing seems to be succesfull. But the led keeps flashing green, indicating that the thingy is crashing.

Custom created pca20020/s132 directory:
![image](https://user-images.githubusercontent.com/56310463/145465082-4a201c7f-8e2f-472c-aee3-284acfcfd379.png)

Any advice on this?

**Comment by Klievan** [comment]
Hi Sam! The ThingySDK is indeed a bit old. Have you tried just using the regular SDK you've been using? The Thingy's run the nRF52832 so you can use those examples as well. You can copy the accelerometer driver over, should be a fairly simple driver.

**Comment by SamGeens** [comment]
Flasing code with PCA10040 and softdevice s132 is working now. I had to completely erase the thingy first with the following command:
` nrfjprog --family nRF52 --eraseall`

The standard Blinky example is running succesfully but our custom code not. The code must be crashing somewhere but we can't find where with the Thingy. 

I've enabled RTT logging and with the standard Blinky example the RTT logs are shown. RTT logs of our custom code on the devkit are working aswell. But this exact same code on the Thingy isn't displaying any RTT logs.
